lib/example_17x_app_web.ex

```diff
@@ -38,7 +38,7 @@ defmodule Example17xAppWeb do
   def controller do
     quote do
       use Phoenix.Controller,
-        formats: [:html, :json],
+        formats: [:html, :json, :"json-api"],
```

Comment:
Based on the mime type in config/config.exs, I appear to be forced to add "json-api" here (because of the dash I need to define it as a stringy atom).